plugin.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <plugin xmlns="http://www.phonegap.com/ns/plugins/1.0"
         id="cordova-plugin-em-opcodeauth"
-        version="1.7.1">
+        version="1.7.2">
```

Comment:
Was this in some way related to the other changes, or was it just a good thing to do while you were here?

Reply:
In general, when we make changes to a native plugin, we create a new release. That version number for the plugin is then "bumped up" in the `package.cordovabuild.json` and the `package.json` of the devapp.